cmd/mattermost/commands/user.go

```diff
@@ -368,6 +379,52 @@ func userCreateCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func usersToBots(command *cobra.Command, args []string, a *app.App) error {
+	if len(args) < 1 {
```

Comment:
Probably we can move this check to the cobra command definition using MinimumNArgs(1) (see here: https://github.com/spf13/cobra#positional-and-custom-arguments)